omod/src/main/webapp/admin/person/include/editPersonInfo.jsp

```diff
@@ -232,7 +232,7 @@
       <td colspan="${fn:length(locales)}">
       <font color="#D0D0D0"><sub>
        <spring:bind path="person.uuid">
-           ${status.value}"
+		   <c:out value="${status.value}"/>
```

Comment:
you seem to be doing a reverse of this ticket https://issues.openmrs.org/browse/LUI-33
I thought all you would need is remove the extra quote.